pkg/config/schema/generate.go

```diff
@@ -0,0 +1,39 @@
+// Copyright 2019 Istio Authors
```

Comment:
Any idea why this isn't showing as a file rename?

Reply:
Maybe because the content changed as well?

Reply:
yeah not sure ... moves often confuse git in interesting ways.